src/component/spinner.js

```diff
@@ -62,11 +63,55 @@ export const LoadNetworkSpinner = ({ percentage, msg, style }) => (
 );
 
 LoadNetworkSpinner.propTypes = {
+  continuous: PropTypes.bool,
   percentage: PropTypes.number.isRequired,
   msg: PropTypes.string.isRequired,
   style: ViewPropTypes.style,
 };
 
+//
+// ContinuousLoadNetworkSpinner
+//
+
+export class ContinuousLoadNetworkSpinner extends Component {
+  constructor(props) {
+    super(props);
+    this.increasePercentage = this.increasePercentage.bind(this);
+    this.state = {
+      percentage: 0,
+    };
+  }
+  componentDidMount() {
```

Comment:
there is currently a prettier plugin being used by eslint https://github.com/lightninglabs/lightning-app/blob/master/.eslintrc
looks like it's just using the default prettier options

Reply:
Relying on default options is dangerous, because people may have different default settings configured in their text editor / IDE. It's more sustainable to provide a .prettierrc in the repo itself.

Reply:
> newlines between functions :)

Agreed. `.prettierrc` is out of scope for this PR. Patches welcome.